### DISCUSSION TITLE
aqualinkd and home assistant

### DISCUSSION THREAD

**Discussion by Kt527** [aqualinkd/AqualinkD#302] [Q&A]
I had aqualinkd running. Now I try implement to home assistant. Since it is my first time on home assistant and I confuse where should I set mqtt_hassio_discover_topic = homeassistant to. Is in aqualinkd.conf or some where else. Thanks for help me out.

**Comment by sfeakes** [comment]
Put that line in AqualinkD.conf.
You will also need to install MQTT broker in home assistant, and configure the other MQTT options in AqualinkD.conf.

**Comment by Kt527** [comment]
Thanks.
